modules/mvt/src/lib/mapbox-vector-tile/vector-tile-feature.d.ts

```diff
@@ -6,10 +6,12 @@ export default class VectorTileFeature {
   properties: object;
   extent: any;
   type: number;
+  id?: number;
 
 	constructor(pbf: Protobuf, end, extent, keys, values);
 
 	loadGeometry(); any;
   bbox(): [number, number, number, number];
-	toGeoJSON(x, y, z): object;
+  toJSON(transform: ([[number]], VectorTileFeature) => void);
+	toGeoJSON(tileIndex: {x: number, y: number, z: number}): object;
```

Comment:
Remove the mix of tabs and spaces?

Reply:
We should get prettier to run on `d.ts`...